doc/source/getting_started/qtcrashcourse.rst

```diff
@@ -91,7 +91,93 @@ http://qt-project.org/doc/qt-4.8/graphicsview.html#the-graphics-view-coordinate-
 Mouse and Keyboard Input
 ------------------------
 
+Events Overview
+^^^^^^^^^^^^^^^
+Understanding events in a Qt application is fundamental before delving into specific input handling such as mouse and keyboard:
 
+- Events in Qt are conceptualized as user interactions with the application, each represented by an event object (QEvent).
+- Various types of events correspond to different user interactions.
+- Event objects encapsulate details concerning the specific occurrence.
+- Dispatched to designated event handlers within the widget where the interaction occurs, these events allow for customizable responses.
+- Handlers may be extended or redefined to modify widget response to interactions.
+
+Mouse Events
+^^^^^^^^^^^^
+Interactions such as clicks, movements, and button releases are managed by overriding methods including ``mousePressEvent``, ``mouseReleaseEvent``, ``mouseDoubleClickEvent``, and ``mouseMoveEvent``.
```

Comment:
These methods should probably link to their upstream methods.   I think the syntax to get the linking to the upstream docs is:

```
:meth:`QtWidget.mousePressEvent <PyQt6.QtWidgets.QtWidget.mousePressEvent>`
:meth:`QtWidget.mouseReleaseEvent <PyQt6.QtWidgets.QtWidget.mouseReleaseEvent>`
:meth:`QtWidget.mouseDoubleClickEvent <PyQt6.QtWidgets.QtWidget.mouseDoubleClickEvent>`
:meth:`QtWidget.mouseMoveEvent <PyQt6.QtWidgets.QtWidget.mouseMoveEvent>`
```